include/swift/Parse/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+  set(SWIFT_GYB_FLAGS --line-directive "^\"#line %(line)d \\\"%(file)s\\\"^\"")
+else()
+  set(SWIFT_GYB_FLAGS --line-directive "\'#line" "%(line)d" "\"%(file)s\"\'")
+endif()
+
+set(generated_include_sources
+  ParsedSyntaxBuilders.h.gyb
+  ParsedSyntaxNodes.h.gyb
+  ParsedSyntaxRecorder.h.gyb
+)
+
+add_gyb_target(swift-parse-syntax-generated-headers
```

Comment:
I think it's safe to add this target to `LLVM_COMMON_DEPENDS` to ensure they are generated prior to any compilation of cpp files. like:
 https://github.com/apple/swift/blob/c932f60403edef804d38b4c29ae99d59ef5eb62b/lib/CMakeLists.txt#L13-L14

Reply:
You have a good point, since those headers are "globally" accessed. I believe we can move all the `Parsed*` and `SyntaxParsingContext.h` headers in `lib/Parse` but until this happens the gyb target should be added to `LLVM_COMMON_DEPENDS` as you suggested.